### DISCUSSION TITLE
No new release since 2020?

### DISCUSSION THREAD

**Discussion by ghost** [ProgerXP/Notepad2e#464] [Questions and feature requests]
I want to try the latest improvement and a 64 bit version. It seems I will have to build from source. Unfortunately for me, I don't have Visual Studio as I only build software using the MSYS2 environment.

**Comment by ProgerXP** [comment]
Indeed I haven't got around to drafting a release because it involves collecting all the updated issues and updating the README. However, I do provide nightly builds so you don't have to compile a thing: https://github.com/ProgerXP/Notepad2e#downloads
